ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -554,6 +568,7 @@ PrjFS_Result PrjFS_DeleteFile(
     
     if (nullptr == relativePath)
     {
+        LogWarning("PrjFS_DeleteFile: relativePath is null");
```

Comment:
Is there some context we could log here? The absolute path and expected prefix perhaps?

Reply:
At this point all we have is `relativePath`.  I've made the message a bit clearer though.